src/components/tooltip/tooltip-styles.jsx

```diff
@@ -150,5 +150,11 @@ export default theme => {
       wordBreak: 'break-word',
       wordWrap: 'break-word',
     },
+    popupDesktopOnly: {
+      display: 'none',
+      [mixins.media('md')]: {
```

Comment:
You can use `maxMedia` here instead so that you don't need to reset the `display`